graftm/db_search_results.py

```diff
@@ -19,6 +19,6 @@ def search_objects(self): # Return SequenceSearchResult object with parameters d
     def hit_count(self): # Return the number of hits found
         return self.hit_count  
     
-    def slash_endings(self): # Return True if sequence headers end in /1 or /2, False if not
-        return self.slash_endings
+    def has_slash_endings(self): # Return True if sequence headers end in /1 or /2, False if not
```

Comment:
well, that may entail some code change, but it just means there is only 1 way to do this, which makes things less confusing. And less code is better, mostly.